modular_skyrat/modules/modular_items/code/bags.dm

```diff
@@ -37,7 +37,6 @@
 	atom_storage.max_total_storage = 12
 	atom_storage.max_slots = 3
 	atom_storage.numerical_stacking = FALSE
-	atom_storage.can_hold = typecacheof(list(/obj/item/ammo_box/magazine, /obj/item/ammo_casing, /obj/item/stock_parts/power_store/cell/microfusion))
```

Comment:
It still neesd to be limited to magazines and casings, no?

Reply:
also did it now